web/modules/custom/collection/src/Form/LeaveCollectionConfirmForm.php

```diff
@@ -113,6 +114,10 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     $memberships = Og::membershipStorage()->loadMultiple($membership_ids);
     Og::membershipStorage()->delete($memberships);
 
+    // @todo: when og gets its own caching system, use its context instead.
+    // Invalidate cache so that the add content block can be rebuilt.
+    Cache::invalidateTags(['block_view']);
```

Comment:
For the same reason as listed above this workaround should be removed. It does not belong here.